denonavr/audyssey.py

```diff
@@ -94,6 +97,10 @@ async def _async_sound_detail_callback(
             self._dynamiceq = "1"
         elif parameter == "DYNEQ OFF":
             self._dynamiceq = "0"
+        elif parameter == "PSLFC ON":
```

Comment:
```suggestion
        elif parameter == "LFC ON":
```
The first 2 characters of the telnet response form the `event` the remaining ones the `parameter`  (see [here](https://github.com/ol-iver/denonavr/blob/0e4f0338a6865759db7c9b110a2e59a1e1fb0dd9/denonavr/api.py#L712-L715)).